mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -70,6 +70,9 @@
 # Maximum time to wait for a response message on the socket.
 _SOCKET_READ_TIMEOUT = callback_handler.MAX_TIMEOUT
 
+# Maximum logging length of Rpc response in DEBUG level.
+_MAX_RPC_RETURN_LENGTH_IN_DEBUG_LOG = 1024
```

Comment:
In debug log is less relevant than verbose logging is off right?
the name of this const should be modified

Reply:
Change to "_MAX_RPC_RESP_LOGGING_LENGTH"

Reply:
I don't see any changes, did you push the commit?

Reply:
Sorry I forgot to commit it before leave the comment. New changes are commit.